plugins/Time/plugin.py

```diff
@@ -206,7 +206,17 @@ def tztime(self, irc, msg, args, timezone):
         irc.reply(datetime.now(timezone).strftime(format))
     tztime = wrap(tztime, ['text'])
 
-
+    def ddate(self, irc, msg, args, year=None, month=None, day=None):
+        """[<year> <month> <day>]
+        Returns a the Discordian date today, or an optional different date."""
+        if year is not None and month is not None and day is not None:
+            try:
+                irc.reply(_ddate(datetime(year=year, month=month, day=day)))
+            except ValueError as e:
+                irc.error("{}".format(e))
```

Comment:
`irc.error("%s", e)`

Reply:
oh right, to maintain backwards compatibility